UI/Selenium/Features/Web Features/ConsultationRoom.feature

```diff
@@ -72,7 +72,7 @@ Scenario: Closed hearings - can join consultation room after hearing is closed
     And all participants leave consultation room
     And everyone signs out
 
-Scenario: Consultation room: Private can start and leave
```

Comment:
What is private?  Is that the private meeting room?

Reply:
I have changed to more descriptive